Cargo.toml

```
@@ -38,8 +38,9 @@ wasm-bindgen = "0.2"
 wasm-bindgen-futures = "0.4"
 
 [features]
-default = ["isahc-static-curl"]
+default = ["isahc-static-curl", "isahc-static-ssl"]
```

What would be the point of using both by default?

```suggestion
default = ["isahc-static-curl"]
```

---

Guessed you aim for a more static-friendly lib. Anyways, fixed.